source/connectors/kafka-docker-example.txt

```diff
@@ -0,0 +1,205 @@
+.. _kafka-docker-example:
+
+======================================
+MongoDB Kafka Connector Docker Example
+======================================
+
+This guide provides an end-to-end setup of MongoDB and Kafka Connect to
+demonstrate the functionality of the MongoDB Kafka Source and Sink
+Connectors.
+
+In this example, we create the following Kafka Connectors:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Connector
+     - Data Source
+     - Destination
+
+   * - Confluent Connector:
+       `Datagen <https://github.com/confluentinc/kafka-connect-datagen>`_
+     - `Avro random generator
+       <https://github.com/confluentinc/avro-random-generator>`_
+     - Kafka topic: pageviews
+
+   * - Sink Connector: **mongo-sink**
+     - Kafka topic: ``pageviews``
+     - MongoDB collection: ``test.pageviews``
+
+   * - Source Connector: **mongo-source**
+     - MongoDB collection: ``test.pageviews``
+     - Kafka topic: ``mongo.test.pageviews``
+
+* The **Datagen Connector** creates random data using the 
+  **Avro random generator** and publishes it to the Kafka topic "pageviews".
+
+* The **mongo-sink** connector reads data from the "pageviews" topic and
+  writes it to MongoDB in the "test.pageviews" collection.
+
+* The **mongo-source** connector produces change events for the
+  "test.pageviews" collection and publishes them to the
+  "mongo.test.pageviews" collection.
+
+Requirements
+------------
+
+Linux/Unix-based OS
+~~~~~~~~~~~~~~~~~~~
+* Docker 18.09 or later
```

Comment:
Can we link to installation instructions? The ones on the docker website worked fine for me on 18.04 https://docs.docker.com/compose/install/

Reply:
Ok sure. Did you have an issue with the version? The original docs mentioned 18.09, so I just used that. If you want to change it to 18.04 and you have confirmed that it works, that's fine with me.